tests/staggeredrenderer/test_staggeredrenderer.cpp

```diff
@@ -89,11 +89,11 @@ void test_StaggeredRenderer::screenToTileCoords_data()
     QTest::addColumn<QPointF>("screenCoords");
     QTest::addColumn<QPointF>("tileCoords");
 
-    QTest::newRow("10,16") << QPointF(10, 16) << QPointF(0, 0);
-    QTest::newRow("5,5")   << QPointF(5, 5)   << QPointF(-1, -1);
+    QTest::newRow("10,16") << QPointF(10, 16) << QPointF(0, 1);
```

Comment:
I think these values are very hard to understand, since they rely on rounding the internal fraction in a coordinate space where rounding makes no sense at all. Please keep these values the same, and change the `test_StaggeredRenderer::screenToTileCoords` to truncate instead of round.